tests/ut/azure_iot_adu_client_ut.c

```diff
@@ -54,7 +54,7 @@ static uint8_t ucSendStateLongPayloadWithRetry[] = "{\"deviceUpdate\":{\"__t\":\
 
 static uint8_t ucSendResponsePayload[] = "{\"deviceUpdate\":{\"__t\":\"c\",\"service\":{\"ac\":200,\"av\":1,\"value\":{}}}}";
 static uint8_t ucHubClientBuffer[ 512 ];
-static uint8_t ucScratchBuffer[ 8000 ];
+static uint8_t ucRequestPayloadCopy[ 8000 ];
```

Comment:
Why is this still necessary?

Reply:
reason given above. copy into just for testing purposes.